docs/riff_streaming_inmemory-gateway.md

```diff
@@ -8,7 +8,7 @@ title: "riff streaming inmemory-gateway"
 
 ### Synopsis
 
-<todo>
+The in-memory Gateway stores messages in memory.
```

Comment:
The CLI never interacts directly with the Gateway resource (except for autocompletion for streams). The resource this command interacts with is InMemoryGateway.

Either lowercase gateway, or use the resource name:

```suggestion
The in-memory gateway stores messages in memory.
```

```suggestion
The InMemoryGateway stores messages in memory.
```

Whichever change is accepted needs to be applied everywhere, and to Kafka and Pulsar as well.

Reply:
went with the first option